packages/site/src/layout/BaseView.tsx

```diff
@@ -37,20 +46,5 @@ BaseView.Siderail = function Siderail({
     return null;
   }
 
-  return (
-    <aside
-      style={{
-        width: "var(--sideBarWidth)",
-        padding: "36px var(--space-base)",
-        display: "flex",
-        flexDirection: "column",
-        flexShrink: "0",
-        boxSizing: "border-box",
-        overflowY: "auto",
-        height: "100%",
-      }}
-    >
-      {children}
-    </aside>
-  );
+  return <aside className="baseView-sideRail">{children}</aside>;
```

Comment:
Moved the styling [here](https://github.com/GetJobber/atlantis/pull/2303/files#diff-6b9fdeb6f7cabf6ded3432d5c06a293ef30380ef98f59cee80fb1b4a3c5c8354R235) to get responsive behaviour.